.travis.yml

```diff
@@ -1,80 +1,67 @@
 # -*- coding: utf-8 -*-
 #
-# This file is part of Invenio
-# Copyright (C) 2015 CERN
+# This file is part of Invenio.
+# Copyright (C) 2016 CERN.
```

Comment:
@JavierDelgadoFernandez Note that good code editors can amend the copyright years for you:
https://github.com/inveniosoftware/invenio/issues/172#issuecomment-42175134